Sources/Algorithms/RandomSample.swift

```diff
@@ -10,8 +10,14 @@
 //===----------------------------------------------------------------------===//
 
 // For log(_:) and root(_:_:)
+#if swift(>=5.11)
+internal import RealModule
+#elseif swift(>=5.10)
+import RealModule
```

Comment:
Importing `RealModule` without `@_implementationOnly` in Swift 5.10, but with `@_implementationOnly` in Swift 5.9 doesn't make sense to me. With `@_implementationOnly` the dependency is hidden from dependents entirely. Without it, the dependency is exposed to dependents. If in practice it is ok to expose the `RealModule` dependency to dependents (as it seems it must be) then it would be more consistent to simply remove `@_implementationOnly` for <= 5.10, rather than having different semantics for 5.10 and <= 5.9.